mobile-app/app/screens/AppNavigator/screens/Balances/hooks/TokenPrice.test.tsx

```diff
@@ -214,37 +220,10 @@ describe('Token Price - Get Token Price (DEX)', () => {
     )
   }
 
-  it('should be able to calculate price rates', () => {
-    const { result } = renderHook(() => useTokenPrice(), { wrapper })
-
-    expect(result.current.calculatePriceRates(
-      'BTC',
-      result.current.getArbitraryPoolPair('BTC', 'USDT'), new BigNumber('2')))
-      .toStrictEqual({
-        aToBPrice: new BigNumber('16600'), // (1000 / 5) * (8300 / 100)
-        bToAPrice: new BigNumber('0.00006024096385542168675'), // (5 / 1000) * (100 / 8300)
-        estimated: new BigNumber('33200') // 2 * (1000 / 5) * (8300 / 100)
-      })
-
-    expect(result.current.calculatePriceRates(
-      'BTC',
-      result.current.getArbitraryPoolPair('BTC', 'USDT'), new BigNumber('1')))
-      .toStrictEqual({
-        aToBPrice: new BigNumber('16600'), // (1000 / 5) * (8300 / 100)
-        bToAPrice: new BigNumber('0.00006024096385542168675'), // (5 / 1000) * (100 / 8300)
-        estimated: new BigNumber('16600') // 1 * (1000 / 5) * (8300 / 100)
-      })
-  })
-
   it('should be able to get the token price', () => {
     const { result } = renderHook(() => useTokenPrice(), { wrapper })
-
-    // DFI / BTC * USDT / DFI (reserve)
-    // (1000 / 5) * (8300 / 100)
-    expect(result.current.getTokenPrice('BTC', new BigNumber('1'), false)).toStrictEqual(new BigNumber('16600'))
-
-    // DFI / ETH * USDT / DFI (reserve)
-    // (1000 / 100000) * (8300 / 100)
-    expect(result.current.getTokenPrice('ETH', new BigNumber('1'), false)).toStrictEqual(new BigNumber('0.83'))
+    expect(result.current.getTokenPrice('BTC', new BigNumber('1'), false)).toStrictEqual(new BigNumber('10000'))
```

Comment:
can you help to add the actual calculation in comment form, I was wondering why is the value is different compared to previous test when the reserve value is the same

Reply:
I'll create a separate ticket for this - https://github.com/DeFiCh/wallet/issues/2634, @fullstackninja864 you can refactor this on another PR